src/Sylius/Bundle/MailerBundle/spec/Form/Type/EmailTranslationTypeSpec.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\MailerBundle\Form\Type;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @author Manuel Gonzalez <mgonyan@gmail.com>
+ */
+class EmailTranslationTypeSpec extends ObjectBehavior
+{
+    function let()
+    {
+        $this->beConstructedWith('ClassName', ['ValidationGroup1, ValidationGroup2']);
```

Comment:
I guess second argument should be an array of two elements, not one. `['ValidationGroup1', 'ValidationGroup2']`